src/CMakeLists.txt

```diff
@@ -439,6 +439,10 @@ if(GL_PROFILE)
     )
 endif(GL_PROFILE)
 
+add_definitions(
+        -DGL_DEBUG
+)
+
```

Comment:
Why?

Reply:
Whoops, I was using that while testing. I'll take it off.